python/src/EnumerateFunctionImplementation_doc.i.in

```diff
@@ -288,3 +288,30 @@ Examples
 %enddef
 %feature("docstring") OT::EnumerateFunctionImplementation::getBasisSizeFromTotalDegree
 OT_EnumerateFunction_getBasisSizeFromTotalDegree_doc
+
+
+// ---------------------------------------------------------------------
+
+%define OT_EnumerateFunction_setUpperBound_doc
+"Accessor to the upper bound.
+
+Parameters
+----------
+ub : sequence of int
+    Upper bound of the indices."
+%enddef
+%feature("docstring") OT::EnumerateFunctionImplementation::setUpperBound
+OT_EnumerateFunction_setUpperBound_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_EnumerateFunction_getUpperBound_doc
+"Accessor to the upper bound.
+
+Returns
+-------
+ub : sequence of int
+    Upper bound of the indices."
```

Comment:
Here we should say that the upper bound is included into the enumeration, e.g. with a bound equal to 4 a polynomial up to degree 4 will be generated.

Reply:
ok, added '(includsive)'